### ISSUE TITLE
[Accessibility] Tooltip does not appear on 'Modify Permissions' menu item with keyboard focus using right/left key.

### ISSUE THREAD

**Issue by ElinorW** [microsoftgraph/microsoft-graph-explorer-v4#1861]
**Test Environment:**
OS Build: Version 22H2(OS Build 25140.1000) 
Browser: Version 104.0.1278.2 (Official build) dev (64-bit)
URL: [Graph Explorer | Try Microsoft Graph APIs - Microsoft Graph](https://developer.microsoft.com/en-us/graph/graph-explorer)

**To Reproduce**
1. Open the above [URL](https://developer.microsoft.com/en-us/graph/graph-explorer) in Edge browser.
2. Navigate to the Request body menu item using tab key.
3. Observe the issue.

**Actual Result:**
Tooltip does not appear on 'Modify Permissions' menu item with keyboard focus using right/left key.
With mouse hover, the user can see the tooltip.

**Expected Result:**
Tooltip should appear on 'Modify Permissions' menu item with keyboard focus using right/left key.

**User Impact:** 
If the user is not able to see the tooltip information as keyboard focus lands on it, the user will not get the additional information present inside the tooltip. Thus, the tooltip should appear with keyboard focus.

**MAS reference:**
[Keyboard - Liquid (microsoft.com)](https://liquid.microsoft.com/Web/Object/Read/MS.Accessibility/Requirements/02.01.01)

**Issue Reference**
https://office.visualstudio.com/OC/_workitems/edit/6179295/

**Comment by ElinorW** [comment]
Hey @RabebOthmani , 
I need some guidance on a this.
I need to remove the tooltips on the Request area tabs and the Response area tabs... But I need to know where we should surface the information on the Modify Permissions tooltip.

The reasons I want to remove the tooltips are because:  
1. It's creating an accessibility bug since the tooltips don't appear when using keyboard navigation.
2. It's something we added, the fluent UI tabs don't actually have tooltips and they don't show any usage of tooltips on tabs. Hence there's no guidance on making them keyboard accessible on tabs.

**Comment by ElinorW** [comment]
This is the information I am referring to 
![image](https://user-images.githubusercontent.com/18407044/179942083-6f857159-12fc-4dc7-89a9-edf2f85834aa.png)

**Comment by RabebOthmani** [comment]
Hi @ElinorW , 
Looking at the Fluent UI guidance, indeed there's no tooltips on the Pivot control. We should always use the Fluent UI components out of the box. The components should be accessible and if not, we should not be customizing them. Instead, we should report the issues to the team to ensure all components are accessible. 
@ddyett @darrelmiller you might be able to help here: Why do we have the preview tag? I'm thinking if we are removing the tag, the info in the tooltip won't be needed anymore.
